hi. There is some trouble in the base_duplicate_security_group. 
See log : 
https://github.com/OCA/server-ux/actions/runs/4893071347/jobs/8735587206?pr=654#step:8:364

```
 2023-05-05 11:56:41,624 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` Whether or not the duplicate button is available The button should be accessible depending on the user permissions. 
2023-05-05 11:56:41,730 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Chrome pid: 921 
2023-05-05 11:56:41,732 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Issuing json command version 
2023-05-05 11:56:41,732 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Url : http://127.0.0.1:52067/json/version 
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Browser version: HeadlessChrome/112.0.5615.121 
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Issuing json command  
2023-05-05 11:56:42,000 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Url : http://127.0.0.1:52067/json 
2023-05-05 11:56:42,004 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Chrome headless temporary user profile dir: /tmp/tmpth_r_o48_chrome_odoo 
2023-05-05 11:56:42,004 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup: Websocket url found: ws://127.0.0.1:52067/devtools/page/86AB1A7FC4765CF2F04CBA491CA9126E 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ERROR 
2023-05-05 11:56:42,283 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ====================================================================== 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ERROR: test_duplicate_button (odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group.TestDuplicateSecurityGroup) 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` Whether or not the duplicate button is available The button should be accessible depending on the user permissions. 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: Traceback (most recent call last): 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/__w/server-ux/server-ux/base_duplicate_security_group/tests/test_base_duplicate_security_group.py", line 21, in test_duplicate_button 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     login="admin", 
2023-05-05 11:56:42,283 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 1083, in browser_js 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.start_browser(self._logger) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 989, in start_browser 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     cls.browser = ChromeBrowser(logger) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 603, in __init__ 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self._open_websocket() 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo/odoo/tests/common.py", line 756, in _open_websocket 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.ws = websocket.create_connection(self.ws_url) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_core.py", line 606, in create_connection 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     websock.connect(url, **options) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_core.py", line 253, in connect 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     self.handshake_response = handshake(self.sock, url, *addrs, **options) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_handshake.py", line 57, in handshake 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     status, resp = _get_resp_headers(sock) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `   File "/opt/odoo-venv/lib/python3.6/site-packages/websocket/_handshake.py", line 148, in _get_resp_headers 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: `     raise WebSocketBadStatusException("Handshake status %d %s", status, status_message, resp_headers) 
2023-05-05 11:56:42,284 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: ` websocket._exceptions.WebSocketBadStatusException: Handshake status 403 Forbidden 
2023-05-05 11:56:42,285 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: Ran 1 test in 0.660s 
2023-05-05 11:56:42,285 890 ERROR odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group: FAILED 
2023-05-05 11:56:42,285 890 INFO odoo odoo.addons.base_duplicate_security_group.tests.test_base_duplicate_security_group:  (errors=1) 
2023-05-05 11:56:42,285 890 ERROR odoo odoo.modules.module: Module base_duplicate_security_group: 0 failures, 1 errors 
2023-05-05 11:56:42,286 890 INFO odoo odoo.modules.module: odoo.addons.base_import_security_group.tests.test_base_import_security_group running tests. 

````
Sometimes it fails, some times it success. 
Two options : 
-  fix the test in  ``base_duplicate_security_group``
- disable the test in ``base_duplicate_security_group``.

@chienandalu, as the maintainer of this module, an idea ? 

thanks !

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -36,9 +36,13 @@ jobs:
       matrix:
         include:
           - container: ghcr.io/oca/oca-ci/py3.6-odoo12.0:latest
+            # See: See: https://github.com/OCA/server-ux/pull/654
+            exclude: "base_import_security_group,base_duplicate_security_group"
             makepot: "true"
             name: test with Odoo
           - container: ghcr.io/oca/oca-ci/py3.6-ocb12.0:latest
+            # See: See: https://github.com/OCA/server-ux/pull/654
+            exclude: "base_import_security_group,base_duplicate_security_group"
             name: test with OCB
     services:
       postgres:
@@ -49,6 +53,9 @@ jobs:
           POSTGRES_DB: odoo
         ports:
           - 5432:5432
+    env:
+      INCLUDE: "${{ matrix.include }}"
+      EXCLUDE: "${{ matrix.exclude }}"
     steps:
       - uses: actions/checkout@v2
         with:

--- a/date_range/__manifest__.py
+++ b/date_range/__manifest__.py
@@ -7,7 +7,7 @@
     "category": "Uncategorized",
     "website": "https://github.com/OCA/server-ux",
     "author": "ACSONE SA/NV, Odoo Community Association (OCA)",
-    "license": "AGPL-3",
+    "license": "LGPL-3",
     "installable": True,
     "depends": [
         "web",

